docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
 * in the Network Boot menu, select the `Dasharo Tools Suite` option,
 * the DTS menu will now appear.
 
+## Common requirements - disabling Secure Boot
```

Comment:
@sulewskiprzemyslaw on booting requirements we say that disabling Secure Boot is needed (here https://github.com/Dasharo/docs/blob/master/docs/common-coreboot-docs/dasharo_tools_suite.md#requirements and here https://github.com/Dasharo/docs/blob/master/docs/common-coreboot-docs/dasharo_tools_suite.md#requirements-1) Lets add link to this section there.

Reply:
OK, link has just been added